Tests/MarathonTests/MarathonTests.swift

```diff
@@ -467,6 +467,92 @@ class MarathonTests: XCTestCase {
+    }
+
+    func testExportingScriptWithName() throws {
+        _ = try makeExportTestFile()
+        try run(with: ["export", "script"])
+        try assertExport(to: folder)
+    }
+
+    func testExportingScriptWithPath() throws {
+        let file = try makeExportTestFile()
+        try run(with: ["export", file.path])
+        try assertExport(to: folder)
+    }
+
+    func testExportingScriptWithExportPath() throws {
+        _ = try makeExportTestFile()
+        let exportFolder = try folder.createSubfolder(named: "temp")
+        try run(with: ["export", "script", exportFolder.path])
+        try assertExport(to: exportFolder)
+    }
+
+    func testExportingScriptWithForce() throws {
+        _ = try makeExportTestFile()
+        try run(with: ["export", "script"])
+
+        let output = try run(with: ["export", "script", "--force"])
+        XCTAssertTrue(output.isEmpty)
+    }
+
+    func testExportingScriptWithDependencies() throws {
+        let file = try folder.createFile(named: "script.swift")
+        try file.write(string: "import Files\n")
+
+        try run(with: ["add", "https://github.com/JohnSundell/Files.git"])
+        try run(with: ["export", "script"])
+
+        let projFolder = try folder.subfolder(named: "script")
+
+        let sourcesFolder = try projFolder.subfolder(named: "Sources")
+        let scriptFileContents = try sourcesFolder.file(named: "script.swift").readAsString()
+        XCTAssertTrue(scriptFileContents == "import Files // marathon:https://github.com/JohnSundell/Files.git\n")
+
+        let packageFile = try projFolder.file(named: "Package.swift")
+        let contents = try packageFile.readAsString()
+        let components = contents.components(separatedBy: .newlines)
+        let dependencyLine = components[5]
+        XCTAssertTrue(dependencyLine.contains(".Package(url: \"https://github.com/JohnSundell/Files.git\", majorVersion: 1)"))
+    }
+
+    private func makeExportTestFile() throws -> File {
+        let file = try folder.createFile(named: "script.swift")
+        try file.write(string: "import Foundation\n")
+        return file
+    }
+
+    private func assertExport(to folder: Folder) throws {
+        let projectFolder = try folder.subfolder(named: "script")
+        let sourcesFolder = try projectFolder.subfolder(named: "Sources")
+        let exportedScriptFile = try sourcesFolder.file(named: "script.swift")
+        let contents = try exportedScriptFile.readAsString()
+        XCTAssertTrue(contents == "import Foundation\n")
+
+        let packageFile = try projectFolder.file(named: "Package.swift")
+        let expected = "import PackageDescription\n"
+            + "\n"
+            + "let package = Package(\n"
+            + "    name: \"script\",\n"
+            + "    dependencies: [\n"
+            + "    ]\n"
+            + ")\n"
+        XCTAssertTrue(try! packageFile.readAsString() == expected)
```

Comment:
Same here, `XCTAssertEquals`. Also, no need to use `try!` since you are already in a throwing method. You can simply use `try`